diem-move/diem-framework/experimental/releases/artifacts/current/build/MoveStdlib/sources/GUID.move

```diff
@@ -21,19 +21,45 @@ module Std::GUID {
         addr: address
     }
 
+    /// A capability to create a privileged identifier on behalf of the given address
+    struct CreateCapability has key, store, drop {
+        addr: address
+    }
```

Comment:
Reason this file is duplicated?

Reply:
I created a release for DPN and experimental, since they both depend on the GUID module. Is it what you're referring to?

Reply:
Yeah